<launch>
	<arg name="model" default='$(find lab3)/urdf/dynamic.urdf' />
	<arg name="gui" default="True" />
	<param name="robot_description" textfile="$(arg model)" />
	<param name="use_gui" value="$(arg gui)"/>
	<param name="arm1" type="double" value="2.0" />
	<param name="arm2" type="double" value="1.0" />
	<param name="joint1_lower" type="double" value="-3.14" />
	<param name="joint1_upper" type="double" value="3.14" />
	<param name="joint2_lower" type="double" value="-2.35619" />
	<param name="joint2_upper" type="double" value="2.35619" />
	<param name="joint3_lower" type="double" value="0" />
	<param name="joint3_upper" type="double" value="3.0" />

	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find lab3)/urdf.rviz" required="true" />
	<node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
	<node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
        <node name="nonkdl_dkin" pkg="lab3" type="nonkdl_dkin" />
	<node name="kdl_dkin" pkg="lab3" type="kdl_dkin" />
	
</launch>
